### PR TITLE
updates to own dactyl keymap

### PR DESCRIPTION
**Pull request by cykedev**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/dactyl_manuform/5x6_5/keymaps/cykedev/rules.mk [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+MOUSEKEY_ENABLE     = no      # Mouse keys
+EXTRAKEY_ENABLE     = yes     # Audio control and System control
+CONSOLE_ENABLE      = no      # Console for debug
+COMMAND_ENABLE      = no      # Commands for debug and configuration
+NKRO_ENABLE         = yes     # USB Nkey Rollover
+BLUETOOTH_ENABLE    = no      # Enable Bluetooth
+AUDIO_ENABLE        = no      # Audio output
+KEY_OVERRIDE_ENABLE = yes     # Enable Key Overrides, see https://docs.qmk.fm/#/feature_key_overrides
+SPLIT_KEYBOARD      = yes     # Split Keyboard
+LTO_ENABLE          = no
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6_5/keymaps/cykedev/rules.mk:17]:
```suggestion
MOUSEKEY_ENABLE = no
KEY_OVERRIDE_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
This has some merge conflicts that need resolution.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
This has a merge conflict that needs to be fixed before we can merge it.

**Comment by cykedev** [comment]
conflict has been resolved

**Pull request merged**